source/macos/AdaptiveCards/AdaptiveCards/Renderers/BaseCardElementRenderer.swift

```diff
@@ -9,8 +9,11 @@ class BaseCardElementRenderer {
     func updateView(view: NSView, element: ACSBaseCardElement, rootView: ACRView, style: ACSContainerStyle, hostConfig: ACSHostConfig, config: RenderConfig, isfirstElement: Bool) -> ACRContentStackView {
         let updatedView = ACRContentStackView(style: style, hostConfig: hostConfig, renderConfig: config)
         
-        // For Spacing
-        if !isfirstElement {
+        if !isfirstElement && element.getSeparator() {
+            // For seperator
+            updatedView.addSeperator(true, withSpacing: element.getSpacing())
+        } else if !isfirstElement {
+            // For Spacing
             updatedView.addSpacing(element.getSpacing())
```

Comment:
why can't we do this?
```suggestion
        if !isfirstElement {
            // For seperator
            updatedView.addSeperator(element.getSeparator(), withSpacing: element.getSpacing())
        }
```

Reply:
Yes, thanks.
done. ✅